server/routes/probationPractitionerReferrals/ppDashboardPresenter.ts

```diff
@@ -7,26 +7,34 @@ import { SortableTableHeaders, SortableTableRow } from '../../utils/viewUtils'
 import PrimaryNavBarPresenter from '../shared/primaryNavBar/primaryNavBarPresenter'
 import LoggedInUser from '../../models/loggedInUser'
 
+export type PPDashboardType = 'Open cases' | 'Unassigned cases' | 'Completed cases' | 'Cancelled cases'
 export default class PPDashboardPresenter {
   constructor(
     private readonly sentReferrals: SentReferral[],
     private readonly interventions: Intervention[],
-    private readonly loggedInUser: LoggedInUser
+    private readonly loggedInUser: LoggedInUser,
+    readonly dashboardType: PPDashboardType
   ) {}
 
-  readonly title = 'Open cases'
+  private readonly showAssignedCaseworkerColumn = this.dashboardType === 'Unassigned cases'
+
+  private readonly dashBoardTypePersistentId = `pp${this.dashboardType.replace(/\s/g, '')}`
```

Comment:
Sorry, another pernickety naming comment here, could we make `dashboard` all lowercase here to keep it consistent with the `dashboardType`?